src-ghc/Pact/GasModel/GasModel.hs

```diff
@@ -0,0 +1,331 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Pact.GasModel.GasModel where
+
+import Control.Exception              (bracket)
```

Comment:
- sort the import list
  - what is the style in pact-land? chainweb is anti-ascii-art/columnar alignment -- I myself am pro :(

I use stylish-haskell to autoformat this stuff in house style -- ping me if you want my .yaml file

Reply:
we're against but it's a recent decision.
sorting is good though

Reply:
I deleted the column alignment in the import list and sorted them in latest commit